pkg/operator/starter.go

```diff
@@ -321,6 +342,8 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	migrationInformer.Start(ctx.Done())
 	apiextensionsInformers.Start(ctx.Done())
 
+	configInformers.WaitForCacheSync(ctx.Done())
```

Comment:
oh, it will block all the other controllers. How about extending the `IsSNOCheckFnc` function to accept `HasSynced` instead?

Reply:
we could simply return `false` before the informer has synced.

Reply:
and then maybe we could log "real" errors in https://github.com/openshift/cluster-openshift-apiserver-operator/pull/489/files#r782969351

Reply:
wdyt ?

Reply:
please remove this line

Reply:
Done